data_summaries/PygmalionAI-PIPPA.json

```diff
@@ -0,0 +1,49 @@
+{
+    "PygmalionAI-PIPPA": {
+        "Unique Dataset Identifier": "PygmalionAI-PIPPA",
+        "Dataset Name": "PIPPA - Personal Interaction Pairs between People and AI",
+        "Paper Title": "PIPPA: A Partially Synthetic Conversational Dataset",
+        "Dataset URL": "https://huggingface.co/datasets/PygmalionAI/PIPPA",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/PygmalionAI/PIPPA",
+        "Papers with Code URL": "https://paperswithcode.com/dataset/pippa",
+        "ArXiv URL": "https://arxiv.org/abs/2308.05884",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "PygmalionAI-PIPPA",
+        "Collection URL": "https://huggingface.co/PygmalionAI",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Conversational Question Answering",
+            "Roleplay"
+        ],
+        "Text Sources": [
+            "https://character.ai"
+        ],
+        "Model Generated": [
+            "Other"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "null"
+        ],
+        "Creators": [
+            "PygmalionAI"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://www.apache.org/licenses/LICENSE-2.0"
+            }
```

Comment:
We may need to add the Character AI terms as well (to here and to the license classes file). cc @rmahari to interpret their terms: https://character.ai/tos

Reply:
This is pending. Can you provide information on what would be most appropriate here?

Reply:
All, 0, 0